React/game/src/app/screens/Login/components/Login/index.js

```diff
@@ -5,17 +5,19 @@ import PropTypes from 'prop-types';
 
 import LoginLayout from './layout';
 
-function Login({ submit, isLogin }) {
-  return <LoginLayout onSubmit={submit} hasError={isLogin} />;
+function Login({ submit, isLogin, msgError }) {
+  return <LoginLayout onSubmit={submit} hasError={isLogin} msgError={msgError} />;
```

Comment:
if `msgError` is not `undefined`, `null` or `empty` then there's an error... also, when `isLogin = false` that doens't mean there's an error, just that the user is not authenticated. Please change this logic

Reply:
👍